packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 0.0.0-internal-20230811175927
```

Comment:
can you remove these internal changes? don't think it should be in the changelog

maybe u can add this to the .changeset file above?